core/primitives/tipset/tipset_key.cpp

```diff
@@ -9,18 +9,30 @@
 #include "common/outcome.hpp"
 
 namespace fc::primitives::tipset {
+  namespace {
+    using ByteArray = std::vector<uint8_t>;
+    outcome::result<ByteArray> cidVectorToBytes(const std::vector<CID> &cids) {
+      ByteArray buffer{};
+      for (auto &c : cids) {
+        OUTCOME_TRY(v, c.toBytes());
+        buffer.insert(buffer.end(), v.begin(), v.end());
+      }
+
+      return buffer;
+    }
+  }  // namespace
 
   std::string TipsetKey::toPrettyString() const {
-    if (cids.empty()) {
+    if (cids_.empty()) {
```

Comment:
minor. this block looks redundant. After removing the output should not be affected

Reply:
It provides fast return when there are no cids, it requires only few machine instructions in the end.